examples/ota-requestor-app/linux/README.md

```diff
@@ -1,108 +1,135 @@
 ```
 
 -   `{REQUESTOR_LONG_DISCRIMINATOR}` is the long discriminator specified for the
-    ota-requestor-app for commissioning discovery
--   `{REQUESTOR_UDP_PORT}` is the UDP port that the ota-requestor-app listens on
-    for secure connections
--   `${PROVIDER_IP_ADDRESS}` is the IP address of the ota-provider-app that has
-    been resolved manually
--   `${PROVIDER_NODE_ID}` is the node ID of the ota-provider-app; this is a Test
-    Mode parameter and should not be used in most scenarios
--   `${PROVIDER_FABRIC_INDEX}` is the fabric index of the ota-provider-app; this
-    is a Test Mode parameter and should not be used in most scenarios
--   `${DELAY_QUERY_SECONDS}` is the amount of time in seconds to wait before
-    initiating secure session establishment and query for software image
-
-In terminal 2:
+    OTA Requestor application for commissioning discovery. If none is supplied,
+    the default is 3840. This must be different from the value used by the OTA
+    Provider application.
+-   `{REQUESTOR_UDP_PORT}` is the UDP port that the OTA Requestor application
+    listens on for secure connections. If none is supplied, the default is 5540.
+    This must be different from the value used by the OTA Provider application.
+-   `{KVS_STORE_LOCATION}` is a location where the KVS items will be stored. If
+    none is supplied, the default is /tmp/chip_kvs. This must be different from
+    the value used by the OTA Provider application.
 
-```
-./chip-tool pairing onnetwork-long ${NODE_ID_TO_ASSIGN_REQUESTOR}  20202021 ${REQUESTOR_LONG_DISCRIMINATOR}
-```
+### In terminal 3:
 
--   `${NODE_ID_TO_ASSIGN_REQUESTOR}` is the node id to assign to the
-    ota-requestor-app running in terminal 3
--   `${REQUESTOR_LONG_DISCRIMINATOR}` is the long discriminator of the
-    ota-requestor-app specified in terminal 3 above
+**Commission the OTA Provider application**
 
 ```
-./chip-tool otasoftwareupdaterequestor announce-ota-provider ${PROVIDER_NODE_ID} 0 0 0 ${REQUESTOR_NODE_ID} 0
+./chip-tool pairing onnetwork-long ${PROVIDER_NODE_ID} 20202021 ${PROVIDER_LONG_DISCRIMINATOR}
 ```
 
--   `${PROVIDER_NODE_ID}` is the node ID of the ota-provider-app assigned to it
-    during the pairing step above
--   `${REQUESTOR_NODE_ID}` is the node ID of the ota-requestor-app assigned to
-    it during the pairing step above
-
-## Note
-
-When the Provider, Requestor and chip-tool are run on the same Linux node the
-user must issue `rm -r /tmp/chip_*` before starting the Provider and
-`rm /tmp/chip_kvs` before starting the Requestor. These commands reset the
-shared Key Value Store to a consistent state.
+-   `${PROVIDER_NODE_ID}` is the node id to assign to the OTA Provider
+    application running in terminal 1
+-   `${PROVIDER_LONG_DISCRIMINATOR}` is the long discriminator of the OTA
+    Provider application specified in terminal 1 above
 
-## Example
```

Comment:
Why remove the example?

Reply:
I reworked the example a bit so it's more generic but still gives you plenty of details